shell.py

```diff
@@ -35,18 +48,16 @@ def show_help_text(self, keyword):
             divider = "-" * len(keyword)
             print(self.help_text.format(**locals()))
         else:
-            print("No help text available for: "+keyword)
+            print("No help text available for: " + keyword)
 
 
 class ListCommands(Command):
 
     keywords = ['commands']
-    help_text = """{keyword}
-{divider}
-Summary: List available commands
-
-Usage: {keyword}
-"""
+    help_text = "{keyword}\n" \
+                "{divider}\n" \
+                "Summary: List available commands\n" \
```

Comment:
I am highly allergic to explicit \n in blocks of text, so I'd like to figure out something better for this. (I admit that having to dedent is gross too.)

Reply:
no problem I will revert it. I just found the dedent more annoying than this 😆